Makefile

```diff
@@ -12,7 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-MKDIR    ?= mkdir
+prefix     ?= /usr/local
+bindir     ?= $(prefix)/bin
+sysconfdir ?= $(prefix)/etc
```

Comment:
Is this standard? I do not see it in the FHS list.

Reply:
Honestly I just copied it from Jon who usually tracks these conventions closely: https://github.com/NVIDIA/enroot/blob/v3.4.0/Makefile#L5

But yes, in this case it does look like it's standard:
> ‘sysconfdir’
> The directory for installing read-only data files that pertain to a single machine–that is to say, files for configuring a host. ...
> https://www.gnu.org/software/make/manual/html_node/Directory-Variables.html

Reply:
I suppose since we're looking at that file I should point out that files like this should really be under `/usr/share/` rather than `/etc/` (see the definition of 'datarootdir'), but I don't personally care to make that breaking change today.

Reply:
The problem with that breaking change is that there are many customers who mount default-counters.csv into /etc/dcgm-exporter/ location via docker -v command-line argument. Such change will break their use cases as they will have to either change where the file is mounted inside the container or change the dcgm-exporter command-line arguments. I would consider both scenarios as a breaking change that needs a major version release at least.

Reply:
Right, yes, I agree. But we're choosing not to make the breaking change, right? I was just making a pedantic point. I'm not arguing the situation change that now or even ever.